app/pages/notifications/notification-section.spec.js

```diff
@@ -27,6 +28,10 @@ const notifications = [
 describe('Notification Section', function() {
   let wrapper;
 
+  before(function () {
```

Comment:
Could you add an after block that restores the class method to its original unwrapped method?